pkg/scrape/target_test.go

```diff
@@ -0,0 +1,149 @@
+package scrape
```

Comment:
apologies, our linting will insist to add a license header here

Reply:
no worries – i just included this in my rebased branch.